diff -urNp -x '*.orig' c_icap_modules-0.5.2.org/configure.ac c_icap_modules-0.5.2/configure.ac
--- c_icap_modules-0.5.2.org/configure.ac	2018-09-13 16:23:05.000000000 +0200
+++ c_icap_modules-0.5.2/configure.ac	2021-10-10 00:11:42.011467022 +0200
@@ -65,7 +65,7 @@ AC_ARG_WITH(c-icap,
 )
 
 if test -z "$cicapdir"; then
-    if c-icap-config >& /dev/null; then
+    if c-icap-config >/dev/null; then
         cicapflags=`c-icap-config --cflags`
         cicaplibs=`c-icap-config --libs`
         cicapetc=`c-icap-config --configdir`
